README.md

```diff
@@ -1,17 +1,22 @@
-<p align="center">
-    <img width="400" height="144" src="/art/logo.png" alt="Graby logo" />
-</p>
+<div align="center">
+    <br>
+    <img width="400" height="144" src="https://user-images.githubusercontent.com/62333/67490348-5dfc5280-f673-11e9-9b3d-584e6cbeb9e2.png" alt="Graby logo" />
+    <br>
+    <br>
+    <br>
+    <br>
+</div>
 
-# Graby
+> **Graby** helps you extract article content from web pages
```

Comment:
I think it makes sense not to duplicate the same paragraph.

Reply:
Good point, didn't notice it was also below badges.